cli/command/service/formatter_test.go

```diff
@@ -95,37 +110,70 @@ bar
 						},
 					},
 				},
+				ServiceStatus: &swarm.ServiceStatus{
+					RunningTasks: 1,
+					DesiredTasks: 3,
+				},
 			},
 			{
-				ID: "id_bar",
+				ID: "02_bar",
 				Spec: swarm.ServiceSpec{
 					Annotations: swarm.Annotations{Name: "bar"},
+					Mode: swarm.ServiceMode{
+						Replicated: &swarm.ReplicatedService{},
+					},
 				},
 				Endpoint: swarm.Endpoint{
 					Ports: []swarm.PortConfig{
 						{
 							PublishMode:   "ingress",
 							PublishedPort: 80,
-							TargetPort:    8080,
-							Protocol:      "tcp",
+							TargetPort:    8090,
```

Comment:
Not an important change, but multiple services cannot public the same port, so this is just making the test-case more realistic/correct.